spec/domain/queries/find_all_document_types_spec.rb

```diff
@@ -0,0 +1,21 @@
+RSpec.describe Queries::FindAllDocumentTypes do
+  context 'when we have data' do
+    before do
+      create :user
+      create :edition, document_type: 'guide'
+      create :edition, document_type: 'travel_advice'
+      create :edition, document_type: 'manual'
+    end
+
+    it 'returns distinct organisations ordered by primary_organisation_title' do
```

Comment:
What about adding an edition that is not latest and checking that it returns the correct value?